src/lemlib/devices/gyro/imu.cpp

```diff
@@ -58,27 +58,27 @@ bool Imu::isCalibrated() { return isConnected() && !imu.is_calibrating() && !std
 bool Imu::isConnected() { return imu.is_installed(); }
 
 /**
- * return the heading of the imu in radians and in standard position
+ * return the heading of the imu in standard position
  */
-float Imu::getHeading() {
-    float heading = std::fmod(getRotation(), 2 * M_PI);
-    if (heading < 0) heading += 2 * M_PI;
+Angle Imu::getHeading() {
+    Angle heading = units::mod(getRotation(), 1_rot);
+    if (heading < 0_deg) heading += 1_rot;
```

Comment:
```suggestion
    if (heading < 0_rad) heading += 1_rot;
```
Change to radians for consistency